cyences_app_for_splunk/default/savedsearches.conf

```diff
@@ -2183,6 +2227,47 @@ action.cyences_notable_event_action.system_compromised_drilldown = `cs_o365` sou
 action.cyences_send_email_action = 1
 action.cyences_notable_event_action.products = Office 365
 
+[O365 - Successful Login From New Location]
+disabled = 1
+enableSched = 1
+alert.track = 1
+alert.severity = 4
+alert.suppress = 0
+counttype = number of events
+quantity = 0
+relation = greater than
+cron_schedule = 59 * * * *
+description = A Successful O365 Login From New Location. \
+\
+Data Collection - Office 365 management activity data (Splunk Add-on for Office 365). 
+dispatch.earliest_time = -2h@h
+dispatch.latest_time = +2h@h
+display.general.type = statistics
+display.page.search.tab = statistics
+display.page.search.mode = fast
+request.ui_dispatch_app = cyences_app_for_splunk
+request.ui_dispatch_view = search
+search = `cs_o365` sourcetype="o365:management:activity" _index_earliest=-61m@m _index_latest=-1m@m Workload=AzureActiveDirectory Operation=UserLoggedIn NOT LogonError=* 
+| stats count, values(user_type) as user_type, values(authentication_method) as authentication_method, max(_time) as Last_Success_Login by user, ClientIP, app 
+| iplocation ClientIP 
+| lookup cs_authentication_usual_location.csv app, user, Country OUTPUT pct 
+| fillnull value=0 pct 
+| where pct < `cs_authentication_new_location_login_pct_limit` 
+| fillnull Country, Region, City value="-" 
+| eval Location=ClientIP." (".count.") | ".Country." | ".Region." | ".City 
+| stats sum(count) as count, values(pct) as pct, values(user_type) as user_type, values(authentication_method) as authentication_method, max(Last_Success_Login) as Last_Success_Login, list(Location) as Location by user 
+| eval cyences_severity = case(user_type=="Admin" or user_type=="DcAdmin", "high", true(), "medium") 
```

Comment:
We need to possibly use pct to apply severity possibly in all of these alerts. Like when original pct is 0 then we can may be keep it critical.